CMakeLists.txt

```diff
@@ -408,18 +408,27 @@ if(UNIX)
   include(CheckSymbolExists)
 
   # don't allow implicit function declarations
-  set(CMAKE_REQUIRED_FLAGS "-std=c99")
-  set(CMAKE_REQUIRED_LIBRARIES "rt")
+  set(CMAKE_REQUIRED_FLAGS "-std=c99 -Werror=implicit-function-declaration")
```

Comment:
Is the -Werror switch something we can expect from ~all compilers and supported versions in UNIX?

Reply:
Good Q. At least clang & gcc should support it, dunno about others.

Reply:
@pjaaskel i'll take care of it in another commit (upcoming PR).